sdk/cosmosdb/cosmos/src/common/platform.ts

```diff
@@ -1,15 +1,31 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { getUserAgent as userAgent } from "universal-user-agent";
+
 import { Constants } from "./constants";
 
 /**
  * @hidden
  */
 export function getUserAgent(suffix?: string): string {
-  const ua = `${userAgent()} ${Constants.SDKName}/${Constants.SDKVersion}`;
+  const ua = `${userAgentDetails()} ${Constants.SDKName}/${Constants.SDKVersion}`;
   if (suffix) {
     return ua + " " + suffix;
   }
   return ua;
 }
+
+/**
+ * @hidden
```

Comment:
`@internal`?

Also this LG, similar enough to https://github.com/gr2m/universal-user-agent/blob/master/index.js so it would be consistent with the existing behaivor

Reply:
Removed as I'm not exporting that function